### ISSUE TITLE
Team-71-the-bulldogs-morning-standup-dec-9th

### ISSUE THREAD

**Issue by codecell** [mariancristu/New-York-Times-Clone#2]

**Comment by srueda27** [comment]
Recommendation: Try not to use to many Id's CSS selectors, better to use class selectors. I think is a better practice

**Comment by codecell** [comment]
yeah just to add to what @srueda27  pointed out, using class selectors also allows for class reusability

**Comment by EdCrux** [comment]
I feel your page will look great. 

Check these styles, I think it is better to use box-sizing as a general rule in all your styles. 

`header {
  width: 800px;
  height: 300px;
  margin: 0 auto;
  box-sizing: border-box;`

**Comment by simonpeterdebbarma** [comment]
# Me looking at your CSS file
<img src="https://media.giphy.com/media/dsc4ptEvModzuAWr7G/giphy.gif"  width="100%">
---

I think it's too early to really say too much, but here's some links to fonts that worked best for me.

- **Libre Franklin** - Top navbar and button elements, use in place of _nyt-franklin_
<link href="https://fonts.googleapis.com/css?family=Libre+Franklin&display=swap" rel="stylesheet"> 
- _IM Fell_ - Page Title, use in place of _nyt-chentalhan_
<link href="https://fonts.googleapis.com/css?family=IM+Fell+English:400i&display=swap" rel="stylesheet">
- Domine - Paragraph elements, use in place of _nyt-imperial_
<link href="https://fonts.googleapis.com/css?family=Domine&display=swap" rel="stylesheet">

**Comment by dxania** [comment]
1. Focusing on your nav bar, good job so far. 
2. A 5px margin to the top on the navbar will look better.
3. The height property within the #nav-middle seems unnecessary. 
4. Don't forget to use clear commit messages as you move forward
5. As mentioned, minimize on the use of id selectors unless your element is distinguishably unique.
